framework/include/kernels/GenericKernel.h

```diff
@@ -0,0 +1,44 @@
+/*************************************************/
+/*           DO NOT MODIFY THIS HEADER           */
+/*                                               */
+/*                     BISON                     */
+/*                                               */
+/*    (c) 2015 Battelle Energy Alliance, LLC     */
+/*            ALL RIGHTS RESERVED                */
+/*                                               */
+/*   Prepared by Battelle Energy Alliance, LLC   */
+/*     Under Contract No. DE-AC07-05ID14517      */
+/*     With the U. S. Department of Energy       */
+/*                                               */
+/*     See COPYRIGHT for full restrictions       */
+/*************************************************/
```

Comment:
This should be a MOOSE header, not a BISON one.

Reply:
> Make sure that your commits are based on top of moose `next` or `devel`. Currently you're adding 4,575 new commits to MOOSE...
> 
> This will also need tests

I'm not sure what happened with that rebase.  I built in BISON's MOOSE and cherrypicked, but it obviously was not right.  Tests are being added within MOOSE.  I had tested in BISON previously